src/nix/flake.md

```diff
@@ -154,6 +156,8 @@ Currently the `type` attribute can be one of the following:
 
   where *path* is an absolute path.
 
+  Non-URI characters in the path are allowed in percent-encoding according to [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986#section-2.1).
```

Comment:
```suggestion
  * The directory path can contain characters that aren't allowed in URIs.
  Those will be handled in [percent-encoding](https://datatracker.ietf.org/doc/html/rfc3986#section-2.1) internally (with the exception of `#` and `?`);
```

If the purpose is to say that the behavior is the same for all path references, we should use the same expression here.